client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -87,9 +86,8 @@ const IntroPricingBanner: FunctionComponent = () => {
 						/>
 						<span>
 							{ preventWidows(
-								translate( '%(days)d day money back guarantee.**', {
+								translate( '%(days)d day money back guarantee.', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.